script.py

```diff
@@ -27,54 +29,64 @@ def generate_token(client_id, client_secret):
         print('Invalid credentials')
 
 
-def request_vehicles(token):
-    return requests.get(f'{osu_API}v1/beaverbus/vehicles', headers={
-        'Authorization': f'Bearer {token}'
-    })
-
-
-def request_vehicle_by_ID(token, ID):
-    return requests.get(f'{osu_API}v1/beaverbus/vehicles/{ID}', headers={
-        'Authorization': f'Bearer {token}'
-    })
+def request_vehicles(*arg):
```

Comment:
`*arg` is a little ambiguous here. You can make this function take two arguments (token and bus ID) and call the corresponding URL depends on if bus ID is null or not.